akka-sample-persistence-dc-java/build.sbt

```
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
```

3.0.7 doesn't seem to be available for scala 2.13?
```suggestion
  "org.scalatest" %% "scalatest" % "3.0.8" % "test"
```